marklogic-data-hub-central/ui/src/config/tiles.config.ts

```diff
@@ -17,46 +17,46 @@ interface TileItem {
 const tiles: Record<TileId, TileItem>  = {
     load: { 
         title: 'Load',
-        iconType: 'fa', 
-        icon: faLongArrowAltRight, 
-        color: '#520339',
-        bgColor: '#F4F6F8',
+        iconType: 'custom',
+        icon: 'loadIcon',
+        color: '#3D409C',
+        bgColor: '#EEE6EB',
```

Comment:
@saarimrahman, looks like you're reverting the gray background in the tile headers back to their earlier color. The bgColor values should all be #F4F6F8 via the following: https://github.com/marklogic/marklogic-data-hub/pull/4635

Reply:
Fixed, thanks